src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/Metrics/IomtMetrics.cs

```diff
@@ -0,0 +1,152 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using EnsureThat;
+using Microsoft.Health.Fhir.Ingest.Telemetry.Metrics;
+
+namespace Microsoft.Health.Fhir.Ingest.Telemetry
+{
+    /// <summary>
+    /// Defines known metrics and metric dimensions for use in Application Insights
+    /// </summary>
+    public static class IomtMetrics
```

Comment:
Feel free to disregard this idea, but I'm wondering if it makes sense to create an IMetric interface that defines the members of a Metric object, and instead of initializing each one of these below, create a separate class for each one of these that implements the IMetric interface. They can all be derived from a common base class and could contain a static method to create a new instance.

Reply:
This is a personal preference, but I like having all of the metrics in one class for now.